.github/CONTRIBUTING.md

```
@@ -41,8 +32,45 @@ Prerequisites:
     outcome of running `tox -e lint` locally is the same as `tox -e lint` being run
     by github actions, you may see the following error: `RuntimeError: failed to
     find interpreter for Builtin discover of python_spec='python3.XX'`. This
-    indicates the version of python that needs to be installed for tox to run
-    locally. In this case, the version of python that needs to be installed is
+    indicates the version of Python that needs to be installed for tox to run
+    locally.
+
+### Linting and formatting
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)